package.nls.ja.json

```diff
@@ -0,0 +1,275 @@
+	"command.removeNewLines.title": "改行を削除",
+	"command.trimAndRemoveNewLines.title": "先頭と末尾の空白をトリミングして改行を削除",
+	"command.replaceWhitespaceWithASingleSpace.title": "連続する空白を半角スペース1つに置換",
+	"command.replaceNewLinesAndWhitespaceWithASingleSpace.title": "連続する空白と改行を半角スペース1つに置換",
+	"command.removeDuplicates.title": "重複する行を削除",
+	"command.caseInsensitiveRemoveDuplicates.title": "重複する行を削除（大文字と小文字を区別）",
+	"command.removeAdjacentDuplicates.title": "隣接して重複する行を削除",
+	"command.caseInsensitiveRemoveAdjacentDuplicates.title": "隣接して重複する行を削除（大文字と小文字を区別）",
+	"command.keepOnlyDuplicates.title": "重複する行以外を削除",
+	"command.caseInsensitiveKeepOnlyDuplicates.title": "重複する行以外を削除（大文字と小文字を区別）",
+	"command.keepOnlyAdjacentDuplicates.title": "隣接して重複する行以外を削除",
+	"command.caseInsensitiveKeepOnlyAdjacentDuplicates.title": "隣接して重複する行以外を削除（大文字と小文字を区別）",
+	"command.keepOnlyUniques.title": "重複する行を全て削除",
+	"command.caseInsensitiveKeepOnlyUniques.title": "重複する行を全て削除（大文字と小文字を区別）",
+	"command.keepNumberOfRandomLines.title": "任意の行数をランダムに抽出",
+	"command.keepPercentageOfRandomLines.title": "割合を指定して行をランダムに抽出",
+	"command.removeEmptyLines.title": "空行を削除",
+	"command.removeSurplusEmptyLines.title": "連続する空行を削除",
+	"command.removeBlankLines.title": "空白行を削除",
+	"command.removeSurplusBlankLines.title": "連続する空白行を削除",
+	"command.removeControlCharacters.title": "制御文字を削除",
+	"submenu.filterLines.label": "行の抽出",
+	"submenu.extraction.label": "行の情報の抽出",
+	"submenu.formatContent.label": "コンテンツのフォーマット",
+	"submenu.changeLetters.label": "文字の変換",
+	"submenu.insertFacts.label": "パスや日時の挿入",
+	"submenu.insertNumbers.label": "連番の挿入",
+	"submenu.insertSequences.label": "連続文字列の挿入",
+	"submenu.paste.label": "展開貼り付け",
+	"submenu.textSlots.label": "テキストスロット（クリップボード）",
+	"submenu.convert.label": "変換",
+	"submenu.increaseDecrease.label": "数値を増やす・減らす",
+	"submenu.encodeDecode.label": "エンコード・デコード",
+	"submenu.joinSplitLines.label": "行の結合・分割",
+	"submenu.sortLines.label": "行の並べ替え",
+	"submenu.generateFakeContent.label": "ダミー・ランダム文字列の生成",
+	"submenu.selection.label": "選択範囲の操作",
+	"submenu.remove.label": "コンテンツの削除",
+	"properties.caseSensitiveFiltering.description": "文字列を抽出する際に大文字と小文字を区別します。",
+	"properties.defaultPadString.description": "「左パディング」「右パディング」コマンドで挿入されるデフォルトの文字。",
+	"properties.insertUppercaseHexNumbers.description": "16進数値を挿入する際に大文字を使用します。",
+	"properties.insertUppercaseGuids.description": "GUIDを挿入する際に大文字を使用します。",
+	"properties.defaultGuidType.description": "GUIDを挿入する際のタイプを制御します。デフォルトでは、コマンド実行時に毎回確認します。",
+	"properties.loremIpsum.wordsPerSentence.min.description": "生成されたLorem ipsumテキストの文中の最小単語数。",
+	"properties.loremIpsum.wordsPerSentence.max.description": "生成されたLorem ipsumテキストの文中の最大単語数。",
+	"properties.loremIpsum.sentencesPerParagraph.min.description": "生成されたLorem ipsumテキストの段落内の最小文章数。",
+	"properties.loremIpsum.sentencesPerParagraph.max.description": "生成されたLorem ipsumテキストの段落内の最大文章数。",
+	"properties.customLocale.description": "日時に関するコマンドを実行する際にシステムロケールの代わりに使用する、カスタムロケール（例えば `en-US`, `hu-HU`）。"

```

Comment:
You should use `ja-JP` instead of `hu-HU` in the Japanese localization, as it is a more proper example for the Japanese audience.